src/Symfony/Component/DomCrawler/Test/Constraint/CrawlerSelectorAttributeValueSame.php

```diff
@@ -47,7 +47,13 @@ protected function matches($crawler): bool
             return false;
         }
 
-        return $this->expectedText === trim($crawler->attr($this->attribute));
+        foreach($crawler->getIterator() as $node){
+            $attrValue = $node->hasAttribute($this->attribute) ? trim($node->getAttribute($this->attribute)) : null;
+            if ($this->expectedText === $attrValue){
+                return true;
+            }
+        }
+        return false;
```

Comment:
```suggestion
        }
       
        return false;
```

Reply:
Thanks for the review, I have already applied the CodingStyle fixes propsed by fabbot in commit  [e7c24c7](https://github.com/symfony/symfony/pull/36062/commits/e7c24c7aaa62694de1f00affdcdd2b8cfc1f1825).